### DISCUSSION TITLE
Do we keep this repo, or move to a new one?

### DISCUSSION THREAD

**Discussion by mauricioszabo** [atom-ide-community/atom#369] [General] [answered]
Many people are asking if there's an "official fork" of the package. Currently, `atom-community` is this fork. Do we keep it, or do we start a new one?

I actually forked myself a new one: https://github.com/AtomRebirth/atom, with the idea of making a monorepo with everything we need to run Atom and not depend on Microsoft's old servers and code anymore. If we decide to keep this repo, I can move my contributions to here, or we can move everything to my org. The important thing is to have a centralized place to this work.

Finally, we need to rebrand things. Unfortunately, Atom (the name and the icon) are trademarks of Microsoft. Some ideas appeared, but we'll need to discuss these soon-ish

**Comment by aminya** [comment, marked as answer]
Please make reviewable pull requests so we can merge them into the repository. Not sure if creating a new organization/repository changes things much. You will miss the already merged contributions, you have to set up the Azure CI again, while I have set it up for this organization. Also, you will miss the members of this organization in the new one. Feel free to remove the icons and brand-related things that you mention.

**Comment by Digitalone1** [comment]
Maybe migrating to Gitlab? It's weird keeping the development of this project on the platform owned by the company that killed it.

**Reply by DeeDeeG** [reply]
A lot of the infrastructure and hard-coded URLs in Atom ("GitHub's code editor") assume GitHub.com is involved somehow. So dropping any dependence on GitHub.com would take work on multiple areas of the code.

Hosting the code on gitlab theoretically wouldn't be an issue. But as it happens, all the energy around Atom is still centered on GitHub at the moment. All of the existing Atom contributors know how to use GitHub.com.

Nothing is stopping anyone of forking and posting the code to their own Gitlab repo. It is open-source, after all. But that can happen any time, and right now whatever minimal amount of developer momentum we have, we need to keep IMO. My personal opinion is toward keeping it on GitHub for now, while acknowledging that yes, **Gitlab is an option**.

**Reply by mauricioszabo** [reply]
@Digitalone1 actually, I made the question when we started to organize ourselves on Atom-Community discord, and I was worried by the lack of activity and the "Sunset" message also on this repo. Since then, things changed A LOT, so I'm not worried anymore about this repo not being used :+1:

**Reply by nembdev** [reply]
> keeping it on GitHub for now

Lets hope this doesn't get forgotten. References in code can be changed. Opensource projects should never feel "trapped" or "forced" to stay on a platform, especially one like github which is solely big business at this point.

When development is in full swing, surely we can make this a priority. We have to vote with our time/presence or we'll lose what little respect/power we have as a community.
